R/dplyr_spark_connection.R

```diff
@@ -20,7 +57,9 @@ sql_translate_env.spark_connection <- function(con) {
       paste0 = function(...) build_sql("CONCAT", list(...)),
       xor = function(x, y) build_sql(x, " ^ ", y),
       or = function(x, y) build_sql(x, " or ", y),
-      and = function(x, y) build_sql(x, " and ", y)
+      and = function(x, y) build_sql(x, " and ", y),
+      pmin = function(...) build_sql_if_compare(..., con = con, compare = " <= "),
```

Comment:
I think the whitespace around the operator here is unnecessary since you call `paste` above with ` and `